python/lsst/ip/diffim/getTemplate.py

```diff
@@ -154,16 +283,21 @@ def run(self, exposure, sensorRef, templateIdList=None):
                 dcrBBox.include(patchInnerBBox)
                 coaddPatch = dcrModel.buildMatchedExposure(bbox=dcrBBox,
                                                            wcs=coaddWcs,
-                                                           visitInfo=exposure.getInfo().getVisitInfo())
+                                                           visitInfo=visitInfo)
             else:
-                if not sensorRef.datasetExists(**patchArgDict):
-                    self.log.warn("%(datasetType)s, tract=%(tract)s, patch=%(patch)s does not exist"
-                                  % patchArgDict)
+                if patchNumber not in availableCoadds:
+                    self.log.warn(f"{patchArgDict['datasetType']}, "
+                                  f"tract={patchArgDict['tract']}, patch={patchNumber} does not exist")
                     continue
-                self.log.info("Reading patch %s" % patchArgDict)
-                coaddPatch = sensorRef.get(**patchArgDict)
+                coaddPatch = availableCoadds[patchNumber]
```

Comment:
Since this is now a dict of data references, this should become:
`coaddPatch = availableCoaddRefs[patchNumber].get()`